Readme.md

```diff
@@ -70,6 +70,10 @@ You can check there's anything to install/upgrade in the `Brewfile` by running:
 
 This provides a successful exit code if everything is up-to-date so is useful for scripting.
 
+For a list of dependencies that are missing, pass `--verbose`.
+
+To check all dependencies by not exiting on the first missing dependency category, pass `--all`.
```

Comment:
I reckon just make this implied by `--verbose` as otherwise just `--all` will produce the same output but take longer to run.